packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -12,7 +12,7 @@ const getStyles = (theme: GrafanaTheme) => ({
   `,
   collapseBody: css`
     label: collapse__body;
-    padding: ${theme.panelPadding};
```

Comment:
@Ijin08 or @dprokop 
> Add padding to Explore panel containers. Graph panel has already had panelPadding, but panelPadding is type of number and therefore it was missing px. Logs panel was missing header padding. Therefore it has been added.

Do you know why there is difference? I see that `panelPadding` is used in computation in one or two places but the inconsistency seems weird. Would it make sense to remove `px` everywhere? Or maybe use `px` but have `stripPx` utility to help with computations?

Reply:
these are special as they are used in computation (the only padding that I know is used this way)

Reply:
Yes I know, but I was wondering if it would not make sense to rather unify the units, to prevent this kind of errors and maybe have a function like `getNumberFromPx(panelPadding) * 2` for computation. Seem like a better deal considering we probably use those units more directly and the computation is an exception.

Reply:
yea, that true. was just nice to avoid parsing/stripping px from strings. With it being a very rare exception maybe just 2 variables is another solution (panelPaddingInt)